app/controllers/api_resource_evaluations_controller.rb

```diff
@@ -1,4 +1,14 @@
 class ApiResourceEvaluationsController < ApiApplicationController
+  def show
+    resource = Resource.find(params[:resource_id])
+    evaluation = resource.resource_evaluations.find_by(user: current_user)
+    if evaluation.nil?
+      handle_record_not_found_error
```

Comment:
The handle_*_error methods doesn't have to be called explicitly. They are called when a exception occurs.

Reply:
Done, using `find_by!` instead of `find_by` the exception is thrown